canopy_dispatch.ml

```diff
@@ -66,67 +66,72 @@ module Make (S: Cohttp_lwt.Server) (C: V1_LWT.CONSOLE) (Disk: V1_LWT.KV_RO)
     | uri::[] when uri = config.Canopy_config.push_hook_path ->
       store.update () >>= fun l ->
       respond_update l
+    | "tags"::[] -> (
+      let tags = Canopy_content.tags !cache in
+      let content = Canopy_article.to_tyxml_tags tags in
+      respond_html ~headers ~title:config.Canopy_config.blog_name ~content ~updated
```

Comment:
this is not very clear to me... atm the cache contains the articles, but no caching of listings or tags is done... maybe they should be cached (as you might have noticed [here](https://hannes.nqsb.io/Posts/nqsbWebsite) I prefer to have as little done as possible on each individual request)?

For this change request here, I'd just use the `store.last_commit () >>= fun updated`, the same used for the atom feed.  Caching can come in a separate PR, but @Engil should tell his opinion :)